xoa/resource_xenorchestra_vm.go

```diff
@@ -127,7 +127,7 @@ func resourceRecord() *schema.Resource {
 						},
 						"mac_address": &schema.Schema{
 							Type:     schema.TypeString,
-							Computed: true,
+							Optional: true,
```

Comment:
We should add `ValidateFunc` here so that we can detect malformed mac addresses. The `jsonrpc2: code -32000 message: unknown error from the peer` error messages are difficult to interpret for most users. The validation should make it clear that the user supplied invalid input instead of them thinking the provider or XO api is not working.